packages/document/main-doc/docs/en/guides/advanced-features/rspack-start.mdx

```diff
@@ -25,27 +25,18 @@ After the project is created, you can experience the project by running `pnpm ru
 :::tip
 When using Rspack as the bundler, the following Features are temporarily unavailable as some of the capabilities are still under development and we will provide support in the future.
 
-- Storybook Devtool
 - The usage of [useLoader](/guides/basic-features/data/data-fetch.html) in Client Side Rendering
 
 :::
 
 ## Enable Rspack build
 
-You can enable Rspack build by running `pnpm run new`:
-
-```bash
-$ pnpm run new
-? Please select the operation you want: Enable features
-? Please select the feature name: Enable Rspack Build
-```
-
-After executing the command, enable the Rspack build in `modern.config.ts`:
+You can enable Rspack build by add the following configuration in `modern.config.ts`:
```

Comment:
We should add a version tip here, users need to upgrade to `@modern-js/app-tools` >= 2.46.0 first.